docs/sample-config.yaml

```diff
@@ -0,0 +1,69 @@
+rules:
+# Each rule represents a some naming and discovery logic.
+# Each rule is executed independently of the others, so
+# take care to avoid overlap.  As an optimization, rules
+# with the same `seriesQuery` but different
+# `name` or `seriesFilters` will use only one query to
+# Prometheus for discovery.
+
+# some of these rules are taken from the "default" configuration, which
+# can be found in pkg/config/default.go
+
+# this rule matches cumulative cAdvisor metrics measured in seconds
+- seriesQuery: '{__name__=~"^container_.*",container_name!="POD",namespace!="",pod_name!=""}'
+  resources:
+    # skip specifying generic resource<->label mappings, and just
+    # attach only pod and namespace resources by mapping label names to group-resources
+    overrides:
+      namespace: {resource: "namespace"},
+      pod_name: {resource: "pod"},
+  # specify that the `container_` and `_seconds_total` suffixes should be removed.
+  # this also introduces an implicit filter on metric family names
+  name: 
+    # we use the value of the capture group implicitly as the API name
+    # we could also explicitly write `as: "$1"`
+    matches: "^container_(.*)_seconds_total$"
+  # specify how to construct a query to fetch samples for a given series
+  # This is a Go template where the `.Series` and `.LabelMatchers` string values
+  # are available, and the delimiters are `${` and `}$` to avoid conflicts with
+  # the prometheus query language
+  metricsQuery: "sum(rate(${.Series}${${.LabelMatchers}$,container_name!="POD"}[2m])) by (${.GroupBy}$)"
```

Comment:
I've been trying to test this PR and tried the example config. I found this line should be `metricsQueries` and not `metricsQuery` based on the value in config.go#L34.

Similarly for the other rules in this config